ui/src/app/views/resource-card/component.tsx

```diff
@@ -481,18 +538,30 @@ export class ResourceCard extends React.Component<ResourceCardProps, ResourceCar
             {fp.startCase(fp.camelCase(this.resourceType.toString()))}</div>
         </div>
       </ResourceCardBase>
-      {this.state.editing && (this.isCohort  || this.isConceptSet) &&
-      <EditModal resource={ResourceCard.castConceptSet(this.props.resourceCard)}
-                 onEdit={v => this.receiveEdit(v)}
-                 onCancel={() => this.closeEditModal()}/>}
+      {this.state.editing && (this.isCohort) &&
```

Comment:
Nit: Can this use `renaming` in state instead? And maybe we can do away with editing?